cmd/agent/main.go

```diff
@@ -116,6 +121,7 @@ func (o *GrpcProxyAgentOptions) Print() {
 	klog.V(1).Infof("SyncInterval set to %v.\n", o.syncInterval)
 	klog.V(1).Infof("ProbeInterval set to %v.\n", o.probeInterval)
 	klog.V(1).Infof("ServiceAccountTokenPath set to %q.\n", o.serviceAccountTokenPath)
+	klog.V(1).Infof("AgentIdentifiers set to %s.\n", o.agentIdentifiers)
```

Comment:
Especially if we decide to encode the data I would suggest we print each key=value pair separately (& decoded).

Reply:
Will print out each identifier separately.